upstream-community-operators/wildfly/wildfly-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,187 @@
+      mediatype: image/png
+  customresourcedefinitions:
+    owned:
+      - name: wildflyservers.wildfly.org
+        version: v1alpha1
+        kind: WildFlyServer
+        displayName: WildFlyServer
+        description: An application running on WildFly application runtime.
+        resources:
+          - kind: Service
+            version: v1
+          - kind: Pod
+            version: v1
+          - kind: StatefulSet
+            version: v1beta2
+        specDescriptors:
+          - description: Number of instances for a WildFlyServer resource.
+            displayName: Size
+            path: size
+          - description: Name of the application image to be deployed.
+            displayName: Application Image
+            path: image
+          - description: spec to specify how standalone configuration can be read from a ConfigMap.
+            displayName: Standalone ConfigMap
+            path: standaloneConfigMap
+          - description: Storage spec to specify how storage should be used.
+            displayName: Storage
+            path: storage
+  install:
+    spec:
+      deployments:
+      - name: wildfly-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: wildfly-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: wildfly-operator
+            spec:
+              containers:
+              - command:
+                - wildfly-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
```

Comment:
Please change this value to `metadata.annotations['olm.targetNamespaces']`

Reply:
fixed